jdaviz/configs/default/plugins/export/export.py

```diff
@@ -290,7 +290,12 @@ def _is_filename_changed(self, event):
     def _update_subset_format_disabled(self):
         new_items = []
         if self.subset.selected is not None:
-            subset = self.app.get_subsets(self.subset.selected)
+            try:
+                subset = self.app.get_subsets(self.subset.selected)
+            except Exception:
+                # subset invalid message will already be set,
+                # no need to set valid/invalid formats.
+                return

```

Comment:
this is needed here because adding support for filtering the select dropdowns meant that this line was getting triggered and raising an exception.  Without this PR, this traceback could still be triggered by going to the export plugin on an invalid subset (e.g., AND logic but not overlapping), but now it was being triggered in the test when importing the subset instead of when accessing it.